src/pages/Loading.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import CircularProgress from '@mui/material/CircularProgress';
+import { Container, Typography } from '@mui/material';
+
+const Loading = () => {
+    return (
+        <Container sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center', minHeight: '100%' }}>
+            <CircularProgress />
+        </Container>
+    );
+}
+
+export default Loading;
```

Comment:
to powinno nazywac się Loader i byc w components, a nie pages. Poza tym uzyj named export a nie default